base/game_events/GameEvent.ts

```diff
@@ -112,16 +112,10 @@ export abstract class GameEvent {
      * @param descriptor the function descriptor.
      * @returns returns a new descriptor.
      */
-    static check_reveal(target: Object, property_key: string, descriptor: PropertyDescriptor) {
-        const original_method = descriptor.value;
-        descriptor.value = function (...args) {
-            const data: GoldenSun = this.data;
-            if (data.hero.on_reveal) {
-                (this.data.info.field_abilities_list.reveal as RevealFieldPsynergy).finish(false, false);
-            }
-            return original_method.apply(this, args);
-        };
-        return descriptor;
+    check_reveal() {
```

Comment:
Please also update the description of this function.

Reply:
Ah!  Right, my bad.  Should be fixed, now.